django_freeradius/tests/base/test_models.py

```diff
@@ -6,6 +6,7 @@
     DEFAULT_SESSION_TIME_LIMIT, DEFAULT_SESSION_TRAFFIC_LIMIT, SESSION_TIME_ATTRIBUTE,
     SESSION_TRAFFIC_ATTRIBUTE,
 )
+from django_freeradius.models import RadiusCheck
```

Comment:
this is what is causing the issue, these tests should not import concrete models because are designed to be reused (and are reused in the sample app).

Read the code of other tests to see how they get the model class: https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/tests/base/test_models.py#L25
You should always do this when in doubt: read how existing code does something.